kernel_gateway/gatewayapp.py

```diff
@@ -291,7 +291,7 @@ def init_webapp(self):
                 default_base_handlers
             ):
                 # Create a new handler pattern rooted at the base_url
-                pattern = url_path_join(self.base_url, handler[0])
+                pattern = url_path_join('/', self.base_url, handler[0])
```

Comment:
If it is already prefixed with a forward slash, this still works, right? (Does test case cover it?)